src/vsr/free_set_encoded.zig

```diff
@@ -101,7 +98,7 @@ pub fn FreeSetEncodedType(comptime Storage: type) type {
         size_transferred: u32 = 0,
 
         // Checksum covering the entire encoded free set.
-        checksum: u128,
+        checksum: u128 = 0,
```

Comment:
`init()` sets `checksum` to `.checksum = vsr.checksum(&.{}),`, but `reset` sets it to the default `0`.